src/features/swap/components/TwapFallbackHandlerWarning/index.tsx

```diff
@@ -0,0 +1,14 @@
+import { Alert, Box, SvgIcon } from '@mui/material'
+import InfoOutlinedIcon from '@/public/images/notifications/info.svg'
+
+export const TwapFallbackHandlerWarning = () => {
+  return (
+    <Box mt={1} mb={1}>
+      <Alert severity="warning" icon={<SvgIcon component={InfoOutlinedIcon} inheritViewBox color="error" />}>
+        <b>Enable TWAPs and submit order .</b>
```

Comment:
```suggestion
        <b>Enable TWAPs and submit order.</b>{' '}
```

Reply:
My IDE actually auto-corrects this to `<b>Enable TWAPs and submit order.</b>  To enable TWAP orders you need to set a custo...` which also seems to work.